pkgs/development/tools/tracy/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, tbb, gtk2, glfw, pkgconfig}:
+stdenv.mkDerivation rec {
+  name    = "tracy-${version}";
+  version = "0.6.3";
+
+  buildInputs = [tbb gtk2 glfw pkgconfig];
+
+  buildPhase = ''
+    make -C profiler/build/unix/ CFLAGS=-ltbb release
```

Comment:
On macOS I get this error

```
In file included from ../../../server/TracyThreadCompress.cpp:3:
../../../server/TracyFileRead.hpp:42:20: error: aligned allocation function of type 'void *(unsigned long, enum std::align_val_t)' is only available on macOS 10.13 or newer
        return f ? new FileRead( f, fn ) : nullptr;
```

You need to add `-faligned-allocation` to `CFLAGS` (on the next line as well) to fix it.

See also #73744.